### DISCUSSION TITLE
Parallel computing

### DISCUSSION THREAD

**Discussion by daniuher** [qMRLab/qMRLab#476] [Q&A]
Hi! Is it possible to run the mono_t2 fit from code on multiple cores via parallel computing please? Thank you in advance!

**Comment by agahkarakuzu** [comment]
Hi @daniuher, yes you can. Just make sure that you are using the qMRLab from the `master` branch, where `ParFitData` is available. You can see how in the documentation below:

https://qmrlab.readthedocs.io/en/master/mono_t2_batch.html#39
